library/billing/src/HCFA_1500.php

```diff
@@ -0,0 +1,771 @@
+<?php
+/*
+ * This program creates the HCFA 1500 claim form.
+ *
+ * @package OpenEMR
+ * @author Rod Roark <rod@sunsetsystems.com>
+ * @author Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2011 Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (C) 2018 Stephen Waite <stephen.waite@cmsvt.com>
+ * @link http://www.open-emr.org
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Billing;
+
+require_once(dirname(__FILE__) . "/../../invoice_summary.inc.php");
+
+use OpenEMR\Billing\Claim;
+use OpenEMR\Billing\HCFA_Info;
+
+class HCFA_1500
+{
+    protected $hcfa_curr_line;
+    protected $hcfa_curr_col;
+    protected $hcfa_data;
+    protected $hcfa_proc_index;
```

Comment:
Note protected is just needed if need to get to them from a class that extends this class (ie. private should be ideal, but we can try changing it back after you get things working first).

Reply:
When things are working, then can change above variables to private

Reply:
When things are working, then can change above variables from protected to private

Reply:
oops, double commented

Reply:
Actually, did some reading on this. Using protected is just fine (just in case anybody ever extends this class).